python/src/rest_api/rest_api_impl.py

```diff
@@ -128,6 +128,6 @@ def _write_openapi_spec_to_file(output_directory: str) -> str:
 
         return f"Successfully wrote the OpenAPI spec to {full_file_path}."
```

Comment:
This return should be moved outside the `try/except`. So, it would be something like
```
try:
     open()
     msg = success
except:
     msg = error
return msg
```
And don't forget you need to update the handling for the command execution for proper execution status. 

So, could alter the write piece to provide msg and status, so something like:
```
try:
     open()
     msg = success
     status = success
except:
     msg = error
     status = failure
return msg, status
```

And then command would be updated to something like:
```
msg_str, status = _write_openapi_spec_to_file()
```

That way you are returning proper execution status of the command and proper message for trying to dump the api to JSON.